docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,163 @@ To get the status of the DNS server:
+<4> Name for the network service (This will appear in the GUI).
+<5> Same as //<4>.
+
+=== Adding the Network Service to _ServiceOrder_ array
+
+Look for the element *<key>ServiceOrder</key>* in the same file, and append the uuid for our _MiniTap_ network service to the array.
+
+----
+<key>ServiceOrder</key>
+    <array>
+        <string>06BFF3C7-13DA-420F-AE9C-B036401184D7</string>
+	<string>58231F56-CA25-4D41-930F-46D83CA07BFE</string>
+	<string>304203B0-AC87-459F-9761-C2799EEBB2E3</string>
+	<string>8655D244-C6E7-4CC0-BF06-BB18F9C3BB85</string>
+	<string>3C26FB9D-D918-4B79-9C7B-ADECD8EFE00F</string>
+	<string>D16F22CE-6DDE-4E63-837C-E16538EA5CCB</string>	//<1>
+    </array>
+----
+
+<1> The uuid for _MiniTap_ network service.
+
+=== Adding the Network Service to _Service_ dictionary
+
+Look for the element *<key>Service</key>* in the same file, and append the following *xml* to its dictionary.
+
+----
+<key>Service</key>
+    <dict>
+        <key>06BFF3C7-13DA-420F-AE9C-B036401184D7</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/06BFF3C7-13DA-420F-AE9C-B036401184D7</string>
+        </dict>
+        <key>304203B0-AC87-459F-9761-C2799EEBB2E3</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/304203B0-AC87-459F-9761-C2799EEBB2E3</string>
+        </dict>
+        <key>3C26FB9D-D918-4B79-9C7B-ADECD8EFE00F</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/3C26FB9D-D918-4B79-9C7B-ADECD8EFE00F</string>
+        </dict>
+        <key>58231F56-CA25-4D41-930F-46D83CA07BFE</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/58231F56-CA25-4D41-930F-46D83CA07BFE</string>
+        </dict>
+        <key>8655D244-C6E7-4CC0-BF06-BB18F9C3BB85</key>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/8655D244-C6E7-4CC0-BF06-BB18F9C3BB85</string>
+        </dict>
+        <key>D16F22CE-6DDE-4E63-837C-E16538EA5CCB</key>					//<1>
+        <dict>
+            <key>__LINK__</key>
+            <string>/NetworkServices/D16F22CE-6DDE-4E63-837C-E16538EA5CCB</string>	//<2>
+        </dict>
+    </dict>
+----
+
+<1> This is the uuid of _MiniTap_ service.
+<2> Replace the uuid with uuid of _MiniTap_ service.
+
+Now reboot your macOS and you should see a _MiniTap_ Service in the Network preferences GUI.
+It will show as disconnected (turned off), to turn it on issue the following commands in a shell.
+
+----
+exec 4<>/dev/tap0			//<1>
+ifconfig tap0 10.10.90.1 255.255.0.0    //<2>
+ifconfig tap0 up			//<3>
+----
+
+<1> Replace it with the `tap` device used in the Network Service.
+<2> IP address should be same as the one in the Network Service definition.
+<3> Replace it with the `tap` device used in the Network Service.
+
+=== Adding resolver config
+
+Lastly create the file `/etc/resolver/nip.io` with the following content.
```

Comment:
"Lastly" is not necessary. :)

Also, please use a colon to end this sentence.

Reply:
Okay :)